rules/windows/network_connection/net_connection_win_notion.yaml

```diff
@@ -0,0 +1,65 @@
+title: Suspicious network connection to Notion API 
+id: 7e9cf7b6-e827-11ed-a05b-15959c120003
+status: experimental
+description: Detects a non-browser process interacting with the Notion API which could indicate the use of a covert C2 such as OffensiveNotion C2
+references:
+    - https://github.com/mttaggart/OffensiveNotion
+    - https://medium.com/@huskyhacks.mk/we-put-a-c2-in-your-notetaking-app-offensivenotion-3e933bace332
+author: Gavin Knapp
+date: 2023/05/03
+tags:
+    - attack.command_and_control
+    - attack.t1102
+logsource:
+    product: windows
+    category: network_connection
+detection:
+    selection:
+        DestinationHostname|contains: 
+            - 'api.notion.com'
+    filter_optional_brave:
+        Image|endswith: '\brave.exe'
+    filter_optional_chrome:
+        Image:
+            - 'C:\Program Files\Google\Chrome\Application\chrome.exe'
+            - 'C:\Program Files (x86)\Google\Chrome\Application\chrome.exe'
+    filter_optional_notion:
+        Image:
+            - 'C:\Users\bwcdemoadmin\AppData\Local\Programs\Notion\Notion.exe'
```

Comment:
Very specific user 😄

Reply:
Damn! Should have waited until the morning to commit. More coffee required 🙃🤣